lib/ansible/modules/messaging/rabbitmq_user.py

```diff
@@ -146,7 +146,7 @@ def _exec(self, args, run_in_check_mode=False):
             if self.node is not None:
                 cmd.extend(['-n', self.node])
             rc, out, err = self.module.run_command(cmd + args, check_rc=True)
-            return out.splitlines()
```

Comment:
I think it's more like:
```python
return out.strip().splitlines()
```
which will have same result

Reply:
indeed